notebooks/semantic_search/cohere_wikipedia_search.ipynb

```diff
@@ -173,15 +173,26 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 10,
+   "cell_type": "markdown",
    "id": "3f8c4efa",
    "metadata": {},
+   "source": [
+    "## Connect to ApertureDB\n",
+    "\n",
+    "For the next part, we need access to a specific ApertureDB instance.\n",
+    "There are several ways to set this up.\n",
+    "The code provided here will accept ApertureDB connection information as a JSON string.\n",
+    "See our [Configuration](https://docs.aperturedata.io/Setup/client/configuration) help page for more options."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fbce988c",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "import os\n",
-    "from getpass import getpass\n",
-    "os.environ['APERTUREDB_JSON'] = getpass()"
+    "! adb config create --from-json --active "
```

Comment:
So we no longer do Colab config as an option? Like - is there a way where you configured Colab to define APERTUREDB_JSON and you could reuse it?

Reply:
The plan here is for the notebook to provide a single way to configure ApertureDB that should work everywhere, with a link to the documentation page that offers other options that might work better in certain contexts.  We don't want to put Colab instructions within every notebook.